findspam.py

```diff
@@ -449,7 +449,8 @@ class FindSpam:
         "onlinecorrection\\.com", "dragracerv3game\\.com", "bananakong\\.net", "himzakaz\\.net", "dropcrack\\.com",
         "raybiztech\\.com", "cegonsoft\\.com", "technomaniya\\.com", "instantassignmenthelp\\.com\\.au", "huintech\\.com",
         "dramaonline\\.pk", "gamingustaad\\.com", "carding-wutransfer\\.com", "nulife\\.co\\.in", "fixmyprinter\\.com",
-        "iseepassword\\.com", "samedaypros\\.com", "zen-promo\\.com", "seyf-educ\\.com",
+        "iseepassword\\.com", "samedaypros\\.com", "zen-promo\\.com", "seyf-educ\\.com", "moneybanker\\.no",
```

Comment:
I'd go with no, pun *fully* intended, on that score.

`.no` is the ccTLD for Norway; I think we should be very careful about blacklisting entire ccTLDs. Country code TLDs are, I'd think, almost always going to have plenty of legit websites.

`.xxx`, on the other hand...